src/preload.ts

```diff
@@ -49,21 +50,19 @@ function attachButtonListeners(): void {
     'nX' // Delete
   ]
 
-  selectors.forEach(async selector => {
-    try {
-      const buttonReady = elementReady(`body.xE .G-atb .${selector}`)
-
-      const readyTimeout = setTimeout(() => {
-        buttonReady.cancel(`Detect button "${selector}" timed out`)
-      }, 10000)
+  selectors.forEach(selector => {
+    const buttonReady = elementReady(`body.xE .G-atb .${selector}`)
+    const readyTimeout = setTimeout(() => buttonReady.stop(), 10000)
 
-      const button = await buttonReady
+    buttonReady.then(button => {
```

Comment:
Why using chaining instead of async/await?

Reply:
After updating deps, it was complaining about returning a non-void value in for each.  The alternative would be to use `.map` but then we are returning an array that is unused.

Reply:
We could use `for` loop instead.

Reply:
After thinking about it, it would probably make more sense to leave it as a `.then` as we don't need to wait for each promise to resolve before attaching the next button listener.